tests/DetectsChangesTest.php

```diff
@@ -33,6 +33,7 @@ public function it_can_store_the_values_when_creating_a_model()
         $expectedChanges = [
             'attributes' => [
                 'name' => 'my name',
+                'text' => null,
```

Comment:
After creating a model, only the attributes that were explicitly set got saved in the activity log. Now, as a result of refreshing the model right after creating it, every attribute in `$logAttributes` will be saved, including the empty ones.